### DISCUSSION TITLE
Is it intentional that on-ride photo sections can now be built underground?

### DISCUSSION THREAD

**Discussion by ocalhoun6** [OpenRCT2/OpenRCT2#18133] [Q&A] [answered]
![Forest Frontiers 2022-09-28 10-07-36](https://user-images.githubusercontent.com/26573303/192845756-3f243527-445b-4686-8a4a-0a01c954f201.png)

Correct me if I'm wrong, but in Vanilla and previously in Open, it wasn't allowed to build on-ride photo sections underground. But now it seems that every ride that has one can build it underground with no problems. I don't recall seeing a PR specifically made to change that ... though maybe I missed it.

I'm just wondering whether this is a bug or an intended feature?

**Comment by IntelOrca** [comment]
Is there a reason not to allow it?
I suppose there is a graphical glitch when placed on the side, the smily face is outside while the track is inside.

**Reply by ocalhoun6** [reply]
Yeah ... not sure why exactly the original game disallowed it, except maybe as a lazy way of fixing that graphical glitch, though that seems unlikely. No IRL reason why a photo section couldn't be placed underground ... except that it might make a boring-looking photo unless the tunnel was well decorated or something.

**Comment by 733737** [comment]
I am in favor of allowing it, but it used to be locked behind cheat codes, it should be discovered what commit caused this since it may have caused other changes as well

**Comment by Gymnasiast** [comment, marked as answer]
It has been allowed for _six years_: https://github.com/OpenRCT2/OpenRCT2/pull/4471

It’s intentional. Lots of RL rides have it underground/in the dark. CS even hacked one into Blackpool Pleasure Beach.

**Reply by ocalhoun6** [reply]
Huh... I guess I just remembered not being allowed to do it, and I never tried for a long time. I only discovered it this time because I did it by accident.

But I guess being a long time ago is why I don't remember seeing the PR for it!
